master/internal/command/command.go

```diff
@@ -278,6 +279,16 @@ func (c *Command) garbageCollect() {
 }
 
 func (c *Command) setNTSCPriority(priority int, forward bool) error {
+	if smallerHigher, err := c.rm.SmallerValueIsHigherPriority(); err == nil {
+		ok, err := configpolicy.PriorityAllowed(int(c.Metadata.WorkspaceID), model.NTSCType, priority, smallerHigher)
```

Comment:
I think we need to be calling 
```suggestion
		ok, err := configpolicy.PriorityUpdateAllowed(int(c.Metadata.WorkspaceID), model.NTSCType, priority, smallerHigher)
```
That function is implemented in the only [commit](https://github.com/determined-ai/determined/pull/10025/files) on main that this branch doesn't have lol 😆. `PriorityAllowed` checks against constraints but we also want to check that this field isn't set in an invariant config, which is done in `findAllowedPriority`, a helper in `PriorityUpdateAllowed`!

Reply:
I changed the name of the function (by request) in a recent PR. Sorry to change the code out from under you.

Reply:
I've changed it. I didn't have any conflicts from git (merge button is still green), it was a semantic conflict. So thanks for the catch.

Reply:
Awesome! Change LGTM, approved!